web/src/popover_menus.js

```diff
@@ -40,6 +40,10 @@ export function popover_items_handle_keyboard(key, $items) {
     let index = $items.index($items.filter(":focus"));
 
     if (key === "enter" && index >= 0 && index < $items.length) {
+        // This is not enough for some elements which need to trigger
+        // natural click for them to work like ClipboardJS and follow
+        // the link for anchor tags. For those elements, we need to
+        // use `.navigate-link-on-enter` class on them.
         $items.eq(index).trigger("click");
```

Comment:
Is the issue here just that the click here goes onto the containing `li`, and not the `a` tag inside?

Reply:
I discussed it in https://chat.zulip.org/#narrow/stream/6-frontend/topic/Navigating.20to.20href.20of.20a.20tags.20in.20popovers. Main problem is `$items.eq(index).trigger("click");` doesn't work like a click by mouse. It doesn't trigger the default browser behaviour of navigating to `href`. or in this case trigger ClipboardJS.

I am tracking it to open a PR to clean up this space so that we won't have to use a workaround like this as suggested by @karlstolley [here](https://chat.zulip.org/#narrow/stream/6-frontend/topic/Navigating.20to.20href.20of.20a.20tags.20in.20popovers/near/1663036)